app/views/data_exports/_manifest.html.erb

```diff
@@ -0,0 +1,68 @@
+<ul class="space-y-4 list-inside list-none mt-2">
+  <% @manifest['children'].each do |project| %>
+   <li>
+      <div class="flex text-primary-800 dark:text-primary-400">
+        <%= viral_icon(name: "folder", classes: "w-5 h-5 m-0") %>
+        <span class="ml-2">
+          <%= link_to project["name"],
+            redirect_from_data_export_path(puid: project["name"]),
+            data: {
+                turbo: false
+            },
+            class: "hover:underline text-sm" %>
+        </span>
+      </div>
+      <% project["children"].each do |sample| %>
+        <ul class="ps-5 mt-3 space-y-1 list-none list-inside">
+          <li>
+            <div class="flex text-purple-800 dark:text-purple-300">
+              <%= viral_icon(name: "folder", classes: "w-5 h-5 m-0") %>
+              <span class="ml-2">
+                <%= link_to sample["name"],
+                redirect_from_data_export_path(puid: sample["name"]),
+                data: {
+                    turbo: false
+                },
+                class: "hover:underline text-sm" %>
+              </span>
+            </div>
+            <% sample["children"].each do |attachments| %>
+              <ul class="ps-5 mt-3 space-y-1 list-none list-inside">
+                <li>
+                  <div class="flex text-blue-800 dark:text-blue-300">
+                    <%= viral_icon(name: "folder", classes: "w-5 h-5 m-0") %>
+                    <span class="ml-2">
+                      <%= link_to attachments["name"],
+                        redirect_from_data_export_path(puid: sample["name"]),
+                        data: {
+                            turbo: false
+                        },
+                        class: "hover:underline text-sm" %>
+                    </span>
+                  </div>
+                  <% attachments["children"].each do |attachment| %>
+                    <ul class="ps-5 mt-3 space-y-1 list-none list-inside">
+                      <li>
+                        <div class="flex text-slate-900 dark:text-white">
+                          <%= viral_icon(name: "document_text", classes: "w-5 h-5 m-0") %>
+                            <span class="ml-2">
+                              <%= link_to attachment["name"],
```

Comment:
I don't actually like the links on the files, they do not take you to the file, but to the sample so just leave them off since you have the link to the sample above.

Reply:
Fixed in 893f42528dd1600d0b22cf05a98d6bc33840d34d